ob_v2p1/ob-spec-v2p1.html

```diff
@@ -807,27 +807,10 @@ <h4>Manifest Properties</h4>
     <section id="api-envelope">
       <h3>Response Envelope</h3>
       <p>
-        Responses from all APIs in this document are wrapped in a JSON envelope. The type of each result in the
-        <code>results</code> array is specified by the result <code>type</code>. Allowable types are defined
-        in the <a href="#api-endpoints">Endpoints</a> section.
+        Responses from all APIs in this document are wrapped in a JSON envelope. All response envelopes have a 
+        <code>status</code> propperty and a <code>results</code> property. The type of each result in the
+        <code>results</code> array is defined in the <a href="#api-endpoints">Endpoints</a> section.
```

Comment:
That's not what we talked about doing, right? There will not be a `results` array in any envelope, but instead each envelope will define one or more properties through which results are delivered. The property names vary depending on the data type of the expected result. (also correct spelling error "propperty")

Updated suggestion for paragraph:
```
Responses from all APIs in this document are wrapped in a JSON envelope. All response envelopes have a 
<code>status</code> property and one or more result properties. The property name(s) and expected
data types for values of each result property is defined in the <a href="#api-endpoints">Endpoints</a> 
section for each endpoint.
```

Reply:
Thank you. Changed.